src/lib.rs

```diff
@@ -274,10 +289,10 @@ where
         Ok(())
     }
 
-    fn await_machine_state(&mut self, target: MachineState) -> Result<(), Error<SpiE>> {
+    fn await_machine_state(&mut self, target_state: MachineState) -> Result<(), Error<SpiE>> {
         loop {
```

Comment:
ugh... must pass some timer thing into the api some day to get rid of this nasty loop {} so stuff can return timeout errors, not relevant here ofc... just a constant reminder.